Iceberg-Plugin-GitHub.package/IceGitHubOpenOnGithubAction.class/instance/basicExecute.st

```diff
@@ -0,0 +1,11 @@
+executing
+basicExecute
+	self class environment
+		at: #WebBrowser
+		ifPresent: [ :webBrowser | 
+			[ | repo |
+			repo := self github getRepository: self organisation project: self projectName.
+			webBrowser openOn: (repo at: 'html_url') ]
+				on: IceGitHubError
+				do: [ self inform: 'Invalid Github repository' ] ]
+		ifAbsent: [ self inform: 'Feature unavailable in Pharo 6' ]
```

Comment:
Can we have a more explicit error? Maybe it can just say "Cannot open {url}: WebBrowser package unavailable" ?
What do you think?

Reply:
Fine with me!